scram/route_manager/api/serializers.py

```diff
@@ -38,7 +38,11 @@ class EntrySerializer(serializers.HyperlinkedModelSerializer):
     )
     route = rest_framework.CidrAddressField()
     actiontype = serializers.CharField(default="block")
-    who = CurrentUserDefault()
```

Comment:
Would it be more or less confusing if this just read:

`who = CurrentUserDefault() or serializes.CharField()`?

Reply:
After looking into this, I find the proposed syntax more confusing (and asked around a little bit to the same response) so for the sake of future us, I'm going to leave the more obvious syntax.